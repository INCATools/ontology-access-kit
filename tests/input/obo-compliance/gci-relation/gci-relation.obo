format-version: 1.4
ontology: gci-relation

[Term]
id: X:1
relationship: R:1 X:2 {gci_predicate="R:2", gci_filler="X:3"}

