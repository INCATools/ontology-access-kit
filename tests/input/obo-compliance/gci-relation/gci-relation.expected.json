{
  "graphs" : [ {
    "id" : "http://purl.obolibrary.org/obo/gci-relation.owl",
    "meta" : {
      "basicPropertyValues" : [ {
        "pred" : "http://www.geneontology.org/formats/oboInOwl#hasOBOFormatVersion",
        "val" : "1.4"
      } ]
    },
    "nodes" : [ {
      "id" : "http://www.geneontology.org/formats/oboInOwl#hasOBOFormatVersion",
      "lbl" : "has_obo_format_version",
      "type" : "PROPERTY"
    }, {
      "id" : "http://www.geneontology.org/formats/oboInOwl#id",
      "lbl" : "id",
      "type" : "PROPERTY"
    } ],
    "edges" : [ {
      "sub" : "http://purl.obolibrary.org/obo/X_1",
      "pred" : "http://purl.obolibrary.org/obo/R_1",
      "obj" : "http://purl.obolibrary.org/obo/X_2",
      "meta" : {
        "basicPropertyValues" : [ {
          "pred" : "http://www.geneontology.org/formats/oboInOwl#gci_predicate",
          "val" : "R:2"
        } ]
      }
    } ]
  } ]
}